cmd/backup-operator/README.md

```diff
@@ -0,0 +1,91 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3].
+
+## Getting Started
+
+You can try out etcd backup operator by running it on `Kubernetes` and then create a `Backup` Custom Resource which contains the targeting etcd cluster and `S3` backup config; the etcd backup operator automatically picks up the `Backup` Custom Resource, retrieves etcd snapshot, and then saves it as a backup to `S3`.
```

Comment:
1. Probably no need to highlight Kubernetes as `Kubernetes`. 
2. Also, we've linked to the Kubernetes page once at the top. Probably no need to link it again in other places down below.